src/silverlabnwb/nwb_file.py

```diff
@@ -72,6 +73,9 @@ def __init__(self, nwb_path, mode='r', verbose=True):
         self.nwb_open_mode = mode
         if mode in {'r', 'r+'} or (mode == 'a' and os.path.isfile(nwb_path)):
             self.open_nwb_file()
+        load_namespaces(
+            'C:\\Users\\Alessandro\\PycharmProjects\\PySilverLabNWB\\src\\silverlabnwb\\silverlab.namespace.yaml')
```

Comment:
This should be made generic; the `__file__` special variable may be a good way to access the location of the namespace file.
Also, maybe this line would be better placed in `add_custom_silverlab_data`? (unless it's needed earlier!)